components/mpas-ocean/src/Registry.xml

```diff
@@ -2,90 +2,90 @@
 <registry model="mpas" core="ocean" core_abbrev="ocn" version="7.0">
 
 	<dims>
-		<dim name="nCells" units="unitless"
+		<dim name="nCells"
 			 description="The number of polygons in the primary grid."
```

Comment:
@darincomeau, yep, this is definitely a desired change.  To comment, you want to go to `Files changed` and use the blue plus sign to highlight one or more lines.  Then, type your comment.

Reply:
There isn't another way to link to the diff as far as I know.  You can link to the file itself.